### PR TITLE
update models and seed

### PR DESCRIPTION
**Pull request by thohuynh**
ready

### CODE REVIEW

**Comment by thohuynh** [comment]
ready

**Comment by thohuynh** [comment]
ready

File: database/seeds/ActivityTableSeeder.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        foreach ((range(1, 20)) as $b) {
+	        DB::table('activities')->insert(
+	            [
```
Comment (trangkarol) [inline database/seeds/ActivityTableSeeder.php]: Convention:
>  DB::table('activities')->insert([
//
]);

File: database/seeds/FollowingTableSeeder.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        foreach ((range(1, 20)) as $b) {
+	        DB::table('followings')->insert(
+	            [
```
Comment (trangkarol) [inline database/seeds/FollowingTableSeeder.php]: Convention:
>  DB::table('activities')->insert([
//
]);

**Comment by trangkarol** [comment]
commented

**Comment by thohuynh** [comment]
ready

**Pull request merged**